product-scenarios/5-Route-messages-between-systems/readme.md

```diff
@@ -0,0 +1,21 @@
+# 5. Route messages between systems
+
+## Business use case narrative
+
+The Message Routing is for reads the content of a message and routes it to a specific recipient based on its content.
+A message router is concerned only with redirecting messages. As an example, if there is a function that pass incoming request on the correct service
+is distributed over to multiple systems, an incoming request needs to be passed on the correct service,
+based on the requested content. In such a scenarios, message router is useful.
+
+Requests going through the Enterprise Integrator are called messages, and message mediation is the main part of an ESB.
+The Switch and filter mediators of the ESB are route the Messages between systems.
+
```

Comment:
Let's explain routing Enterprise integration pattern with diagram and add the reference to that relevant doc

[1] https://docs.wso2.com/display/IntegrationPatterns/Message+Router